docs/agent-api.asciidoc

```diff
@@ -68,16 +68,22 @@ TIP: Before using custom context, ensure you understand the different types of
 [[apm-add-tags]]
 ==== `apm.addTags()`
 
+deprecated, Please use <<apm-add-labels,`apm.addLabels()`>> instead
```

Comment:
I think we should add a little more clarification here. We also have deprecated tags, the format is: `deprecated[x.x.x]`. Something like:
> deprecated[1.2] Tags have been replaced with labels. Please use <<apm-add-labels,`apm.addLabels()`>> instead.

Reply:
Nice, Will change it accordingly :)